lib/statsd.js

```diff
@@ -464,5 +462,37 @@ Client.prototype.childClient = function(options) {
   return new ChildClient(this, options);
 };
 
+// Overrides tags in parent with tags from child with the same name (case sensitive) and return the result as new
+// array. parent and child are not mutated.
```

Comment:
Nitpick, but can you change this to be a /* */ style comment like the other function-level comments?